### PR TITLE
F3 02 user selects alternatives

### PR DESCRIPTION
**Pull request by elaguerta**
Having issues with passing the ingredient alternatives to the `recipe/view/html` template. See comments in-line.

### CODE REVIEW

**Comment by scheffeltravis** [comment]
Looks like you included your other changes in this PR so you will either need to fix conflicts here when the other is merged, or delete the other and simply work from this one.

File: example/flaskr/ingredient.py [hunk 1]
```diff
@@ -65,7 +70,6 @@ def getAlternatives(ingredientName):
     return render_template("recipe/ingredient.html", ingredients=alts)
 
 
-@bp.route("/<ingredientName>/alt/<ratingName>")
```
Comment (scheffeltravis) [inline example/flaskr/ingredient.py:72]: Glad to see we're getting rid of these extra pages as I was unsure if people would actually be navigating to them.

**Review by scheffeltravis — APPROVED** [review]

**Comment by elaguerta** [comment]
@CHRYSICS Here is what `/ingredient/<ingredientName>` looks like now. If this looks good, please approve and I'll merge this in and hand it off to you to turn it into a form!
image

File: example/flaskr/ingredient.py (hunk 1 above)
Comment (CHRYSICS) [reply example/flaskr/ingredient.py:72]: Yep, this is a good code fix!

**Review by CHRYSICS — APPROVED** [review]
This looks good and ran as expected from your image! Thanks Elaine!

**Pull request merged**